main.go

```diff
@@ -734,10 +741,50 @@ func commentToNotifyUser(args commentArgs) error {
 	if !args.mergeConflicts {
 		commentBody = fmt.Sprintf("@%s I have created a PR for you, ready to merge as soon as tests are passed", args.userName)
 	} else {
-		msg := "@%s I have created a PR for you. Unfortunately there were some merge conflicts " +
-			"which I failed to resolve automatically. Therefore the branch of this PR " +
-			"is ready to be merged locally, and then pushed here :)"
-		commentBody = fmt.Sprintf(msg, args.userName)
+		tmplString := `\
+@{{.UserName}} I have created a PR for you.
+Unfortunately, a merge conflict was detected. This means that the conflict will have to be resolved manually by you, human. Then pushed to the PR-branch, once all conflicts are resolved.
+This can be done by following this recipe:
+
+* Make sure that the 'mender-test-bot' remote must be present in your repository, if not:
```

Comment:
Nitpick: "make sure that [..] must be present" -> "Make sure that [...] is present" or "[...] must be present"

Reply:
Great! Thanks, I like it.

Fixing